Signal/src/AppDelegate.m

```diff
@@ -819,6 +817,16 @@ - (void)databaseViewRegistrationComplete
 
     if ([TSAccountManager isRegistered]) {
         DDLogInfo(@"localNumber: %@", [TSAccountManager localNumber]);
+
+        // Fetch messages as soon as possible after launching. In particular, when
+        // launching from the background, without this, we end up waiting some extra
+        // seconds before receiving an actionable push notification.
+        [[Environment getCurrent].messageFetcherJob runAsync];
+
+        // This should happen at any launch, background or foreground.
+        __unused AnyPromise *promise = [OWSSyncPushTokensJob runWithPushManager:[PushManager sharedManager]
```

Comment:
why assign to a local variable?

Reply:
It's the simplest way, in tandem with the __unused macro to avoid a warning here. Is there a better way?